Sources/PrimerSDK/Classes/Data Models/PrimerContent.swift

```diff
@@ -208,7 +208,7 @@ struct CardFormViewContent {
     var cvcTextFieldPlaceholder: String { return NSLocalizedString("primer-card-form-cvv",
                                                                    tableName: nil,
                                                                    bundle: Bundle.primerResources,
-                                                                   value: "CVV",
+                                                                   value: "CVV/CVC",
```

Comment:
Can you double check with a designer on this?

Reply:
With Tim, we decided to keep it as it is right now: `CVV`.